Models/reviews.models.js

```diff
@@ -1,14 +1,19 @@
 const db = require("../db/connection.js");
 
 exports.fetchReview = (review_id) => {
-    return db.query("SELECT * FROM reviews WHERE review_id = $1", [review_id])
-    .then((result) => {
-      const review = result.rows
+    const review = db.query("SELECT * FROM reviews WHERE review_id = $1", [review_id])
```

Comment:
Refactor this into a single query. Looking at using COUNT(), LEFT JOIN and GROUP BY, to count the comments that have a relationship with the article that you're requesting